apps/api/deps/auth.py

```
@@ -22,13 +22,11 @@ async def get_redis_client(request: Request) -> Redis:
     Returns:
         redis.Redis[bytes]: Async Redis client (returns bytes, not decoded strings).
     """
-    from typing import cast
-
-    return cast(Redis, request.app.state.redis)
+    return cast(Redis[bytes], request.app.state.redis)
 
 
 async def verify_api_key(
-    redis_client: Annotated[Redis, Depends(get_redis_client)],  # noqa: B008
+    redis_client: Annotated[Redis[bytes], Depends(get_redis_client)],  # noqa: B008
```

Ruff (0.14.1)

Unused `noqa` directive (unused: `B008`)

Remove unused `noqa` directive

(RUF100)